newVersion.py

```diff
@@ -155,25 +163,44 @@ def modify_package_resolved(args, filename):
             package = match.group(1)
         if package == 'itwin-mobile-native':
             line = re.sub('("version": )".*"', '\\1"' + args.new_add_on + '"', line)
-            if (args.new_add_on_commit_id):
+            if hasattr(args, 'new_add_on_commit_id'):
```

Comment:
Should you also check that the value is non-null?

Reply:
I'm not sure, possibly. How would it get a null value if it has the attribute?

Reply:
I'm worried that if the old code worked before your changes, that your new code won't work. I ran into that problem when I made a similar change. I had to update it to look like this:

    if not hasattr(args, 'new_mobile') or not args.new_mobile:

Reply:
Fixed